commit d81d737495a1b4da4f85013e95a86848ca290fb9 (refs/changes/36/4036/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-30T21:40:32-07:00 (1 year, 4 months ago)
    
    TRITON-380 work around logadm.conf "/var/log/*.log" entry in newer origins to not lose logs

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index bda632a..6bfa4a2 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
+Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
